### ISSUE TITLE
Gadam or gadām?

### ISSUE THREAD

**Issue by MagdaKrzyz** [BetaMasaheft/Documentation#2136]
https://betamasaheft.eu/authority-files/list?keyword=gadam

**Comment by MagdaKrzyz** [comment]
I can see the capitalization in the list in Dillmann's Lexicon, in the menu.
![Screenshot 2022-05-24 122715](https://user-images.githubusercontent.com/29131132/170011335-993de338-4f1f-4b13-833f-38d90250e0e0.png)

**Comment by eu-genia** [comment]
I see, just the first link you posted was to BM not to Dillmann, there must be some other taxonomy there

**Comment by MagdaKrzyz** [comment]
Ok, I though this is the same list.

**Comment by eu-genia** [comment]
no, the two apps are not connected, this list is in Dillmann guidelines, fixed

**Comment by MagdaKrzyz** [comment]
No, it is not in Dillmann's guidlines. I have not provided the list either.

**Comment by eu-genia** [comment]
can you check please? it is the file called `guidelines.xml`, fixed

**Comment by MagdaKrzyz** [comment]
I cannot check that because I have no access to xml files. I asked Denis and he will make the corrections. 
I use and develop the guidlines in Wiki : https://github.com/BetaMasaheft/Dillmann/wiki

**Comment by eu-genia** [comment]
I think there is a lot of misunderstanding going on.

So you still see the errors you were complaining of?

Can you please check and post the screenshot?
